### ISSUE TITLE
Warnings with the R create_results() command line

### ISSUE THREAD

**Issue by SimonBuatois** [pharmpy/pharmpy#135]
Hi,

When reading results from bootstrap or Frem tools,  I have the following warning messages:
![image](https://user-images.githubusercontent.com/75624013/114831080-44058100-9dcd-11eb-9e95-14929c8ce9b7.png)

This is true regardless of the tool or the run.

Best,
Simon

**Comment by rikardn** [comment]
Thanks for reporting this. Could you please check which version of pandas you are using with:

```R
library(reticulate)
pd <- import("pandas")
pd$`__version__`
```

**Comment by SimonBuatois** [comment]
> library(reticulate)
> pd <- import("pandas")
> pd$`__version__`
[1] "1.2.0"

**Comment by rikardn** [comment]
Thanks! The should be fixed in the next Pharmpy release (0.23)
